docs/release-notes.md

```diff
@@ -32,6 +32,21 @@ updates as well.
   2.9.10 has an infinite loop in a certain end-of-file situation." Fix
   by updating a dependency in documentation generation.
 
+### Zephyr-Rtos compatibility
+- for compatibility with zephyr 2.2.0 need to revert following commits:  
```

Comment:
`for compatibility with zephyr 2.2.0 the following commits need to be reverted:`